Server/string-decoder.ts

```diff
@@ -0,0 +1,6 @@
+class StringDecoder {
```

Comment:
For NodeJS, we want to make things exportable. To do that, just tack on export before class:
`export class StringDecoder`